### PR TITLE
Fixes sprint 2017 Week 24-25

### PR DESCRIPTION
**Pull request by dgutierr**
Fixes + tests

@romartin Can you approve & merge please?

### CODE REVIEW

File: dashbuilder-client/dashbuilder-common-client/pom.xml [hunk 1]
```diff
@@ -54,6 +54,16 @@
     </dependency>
 
     <dependency>
+      <groupId>org.jboss.errai</groupId>
```
Comment (romartin) [inline dashbuilder-client/dashbuilder-common-client/pom.xml]: I think it's missing to add the GWT modules dependencies for `errai-common` & `errai-ui` into `dashbuider-common-client` module as well

**Comment by dgutierr** [comment]
jenkins retest this

File: dashbuilder-client/dashbuilder-common-client/pom.xml (hunk 1 above)
Comment (dgutierr) [inline dashbuilder-client/dashbuilder-common-client/pom.xml]: Done.

**Comment by romartin** [comment]
Hey @dgutierr 
Running this on wildlfly results in several warnings on startup (about errai), that should not be there... in addition, from the UI, once clicking on `Gallery`, the whole main menu disappears too.... I think something is not right yet.

screenshot from 2017-06-21 19-05-11

**Comment by dgutierr** [comment]
@Roger both issues are fixed now.

The Errai warnings were caused by a client side dep not excluded from WAR.

Please have a look. Thanks!

**Comment by romartin** [comment]
Merged:
04c315032f302554a6bfd064d7399bf8521942dc
c48aa563c11911e40d76f59ecdaab50291fd6f90
258510f2a1843b9f549ef32cbed45caafe91448c

**Pull request closed without merging**